app-main/public/js/index.js

```diff
@@ -11,6 +11,9 @@ const avatars = [
   "octopus",
   "parrot",
 ];
+if (outerWidth < 400) {
```

Comment:
Do we still need this code? Can we remove it?

Reply:
Oops, I'll remove this too.🙂